i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/users/README.md

```diff
@@ -79,3 +79,93 @@ _用户_ 是身份服务的主要实体。我们将会在下文描述用户相
 你可以使用「[管理控制台](../../../docs/recipes/manage-users/using-admin-console#查看和更新用户资料)」或
 「[管理 API](../../../docs/recipes/manage-users/using-management-api)」来查询用户资料，
 例如 <a href="/api/#tag/Users/paths/~1api~1users~1:userId/get" target="_blank">`GET /api/users/:userId`</a>。
+
+### basic_data
+
+用户的基础数据，即在预定义的属性中保存的用户基础信息。
+
+#### user_id
+
+user_id 即用户 ID，是自动生成的唯一键，用于标识 Logto 用户。
+
+#### username
+
+username 即用户名，用于用户名加密码的登录方式。
+
+如果用户没用过用户名加密码的方式注册，他们的 username 可能是空的。
+username 的值非空时，最多 128 个字符，只能包含字母、数字和下划线（`_`），不能以数字开头。
+
+#### primary_email
+
+primary_email 即主要邮箱，是用户的电子邮件地址，用于邮箱加验证码的登录方式。
+
+如果用户没用过邮箱注册，他们的 primary_email 的值可能是空的。
+primary_email 的值非空时，最多 128 个字符。
+
+#### primary_phone
+
+primary_phone 即主要手机号，用于手机号加短信验证码的登录方式。
+
+如果用户没用过手机号注册时，他们的 primary_phone 的值可能是空的。
+primary_phone 的值非空时，只能包含数字，且以 [国家呼叫代码](https://en.wikipedia.org/wiki/List_of_country_calling_codes) 为前缀（加号 `+` 除外）。
+
+#### name
+
+name 即用户的全名，最多 128 个字符。
+
+#### avatar
+
+avatar 即用户的头像，是指向用户头像图片的 URL。
+
+如果用户使用 Facebook 和微信等社交连接器注册，则可能会直接采用社交用户信息中获取到的头像。
+允许的图片 URL 最多 2048 个字符。
+
+#### role_names
+
+roles_names 即用户的角色名称集合，表示在 Logto 中授予给该用户的角色集合。
+
+role 即角色，代表你授予给用户的一组权限。
```

Comment:
"role" should not appear in this section.

Reply:
removed this paragraph.